nsot/models/attribute.py

```diff
@@ -157,7 +151,7 @@ def clean_fields(self, exclude=None):
         self.resource_name = self.clean_resource_name(self.resource_name)
         self.name = self.clean_name(self.name)
 
-    def _get_model_cls(self):
+    def get_resource_class(self):
```

Comment:
Now that we've solidified this, can you please ditch the `TODO` comment and give this a simple docstring?

Reply:
Done.